targets/targets.json

```diff
@@ -443,6 +443,7 @@
         "supported_toolchains": ["ARM", "GCC_ARM", "IAR"],
         "inherits": ["Target"],
         "detect_code": ["0200"],
+        "macros": ["MBED_DISABLE_ISR_LOGGING=1"],
```

Comment:
I do not understand why we do use macro likes this still. How to redefine this in the application? macro_remove, macro add? isn't it easier via config  (overwrite the value that is it) ?

Reply:
I can have this macro as CONF, but this has to be added to json file for specific targets. Will that be fine?

Reply:
Can't you use the "macro_name" attribute for config variables to not add the CONF_BLAH part?